### PR TITLE
9-11 Black Jack - Teresa

### PR DESCRIPTION
**Pull request by terenex**
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
5 hours

Please fill in one error and/or error message you received while working on this assignment.
Had a lot of undefined error messages, but realized it's due to where i was enclosing my codes (within the {} )

What part of the assignment did you spend the most time on?
The second and third version

Comfort Level (1-5):
3

Completeness Level (1-5):
3

What did you think of this deliverable?
Tough! 

Is there anything in this code that you feel pleased about?
Pleased at my logic and how I got it to work on my own, albeit in a very long winded manner.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,4 +1,256 @@
...
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  // initialize output
+
+  var output = "";
+
+  //pop takes the card at the end of the array and removes
+
+  var computerCard1 = shuffledDeck.pop();
```
Comment (michelle-mok) [inline script.js:116]: consider using a function (containing a loop) to generate player/ computer's initial hands

File: script.js [hunk 2]
```diff
@@ -1,4 +1,256 @@
...
+    } and ${computerCard2.name} of ${computerCard2.suit} .<br>You drew ${
+      playerCard1.name
+    } of ${playerCard1.suit} and ${playerCard2.name} of ${
+      playerCard2.suit
+    }.<br><br>The computer scored ${computerScore()}, while you scored ${playerScore()}. `;
+
+    output = messageShuffledDeck + messageHitStand;
+    gameMode = "hit or stand";
+
+    if (playerScore() < 17) {
```
Comment (michelle-mok) [inline script.js:160]: consider storing player's score in a variable, eg,` var playerScore = ....`

**Review by michelle-mok — COMMENTED** [review]
good use of game modes and functions. 
consider storing player/ computer hands in an array, eg,` var playerHand = []`. this way you can have multiple cards without having to name them (playerCard1, playerCard2 etc...)

**Comment by terenex** [comment]
Please fill out the survey before submitting the pull request. Thanks!

🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀🚀

How many hours did you spend on this assignment?
5+5 hours (to complete the rest of the versions) = 10 hours

Please fill in one error and/or error message you received while working on this assignment.
Had a lot of undefined error messages, learned that it's due to not returning output at the right areas

What part of the assignment did you spend the most time on?
The second and third version

Comfort Level (1-5):
2

Completeness Level (1-5):
5

What did you think of this deliverable?
Tough! Wouldn't have known what to change if not for the blackjack redux class. 

Is there anything in this code that you feel pleased about?
Pleased at my logic and how I got it to work on my own, albeit in a very long winded manner.

**Pull request closed without merging**